### PR TITLE
fix: deprecated plugin, marking it compat <= 24.2 (#260)

### LINKED ISSUE #260: deprecate intelllij-knative
We dont have the people to work on this plugin. It is [not compatible](https://plugins.jetbrains.com/plugin/16476-knative--serverless-functions-by-red-hat/edit/versions/stable/526123) with `IU-2024.3` any more. We're thus deprecating this plugin. 

image

We're moving the functionalities i[ntellij-openshift-connector](https://github.com/redhat-developer/intellij-openshift-connector/) though. You can track progress here:  https://github.com/redhat-developer/intellij-openshift-connector/issues/887

### PR DESCRIPTION
**Pull request by adietish**
fixes #260

### CODE REVIEW

**Comment by adietish** [comment]
@lgrossma, @sbouchet: added a comment in README and set the upper bound of the compatibility range (`until-build="242.*"`). Missing something?

File: .github/workflows/IJ.yml [hunk 1]
```diff
@@ -17,9 +17,9 @@ jobs:
         IJ: [IU-2022.3, IU-2023.1, IU-2023.2, IU-2023.3, IU-2024.1]
```
Comment (sbouchet) [inline .github/workflows/IJ.yml:17]: just spotted that we're missing IJ-2024.2 here .
```suggestion
        IJ: [IU-2022.3, IU-2023.1, IU-2023.2, IU-2023.3, IU-2024.1, IU-2024.2]
```

**Review by sbouchet — COMMENTED** [review]
add the missing IJ compatible version in workflow.

**Comment by adietish** [comment]
@sbouchet: my bad, the correct upper bound should be `2024.1`, not `2024.2`

**Review by sbouchet — APPROVED** [review]
LGTM.

**Pull request merged**